pkg/kubernetes/kubernetes_test.go

```diff
@@ -32,7 +32,7 @@ func TestGetPods(t *testing.T) {
 	k.Client.CoreV1().Pods(expected[0]).Create(context.TODO(), pod, metav1.CreateOptions{})
 
 	// execute
-	actual := GetPods(expected[0])
+	actual := GetPods(context.TODO(), expected[0])
```

Comment:
@inkel - Is there a better way to handle contexts inside tests? 🤔

Reply:
You mean [this](https://pkg.go.dev/testing#T.Context)? 😬 
```suggestion
	actual := GetPods(t.Context(), expected[0])
```